main/icicle-repl.hs

```diff
@@ -39,7 +39,7 @@ runRepl
     loop state
       = do line <- HL.getInputLine "> "
            case line of
-             Nothing      -> loop state
+             Nothing      -> return ()
```

Comment:
cool, I'll leave CommandBlank in then.

see https://github.com/ambiata/icicle/pull/32